src/frontend/screens/Track/index.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  SafeAreaView,
+  TouchableOpacity,
+} from 'react-native';
+import {BLACK, DARK_GREY} from '../../lib/styles.ts';
+
+import TrackIcon from '../../images/Track.svg';
+import EditIcon from '../../images/Edit.svg';
+import {FormattedMessage, MessageDescriptor, defineMessages} from 'react-intl';
+import {useDeleteTrackMutation, useTrack} from '../../hooks/server/track.ts';
+import {useObservations} from '../../hooks/server/observations.ts';
+import {NativeNavigationComponent} from '../../sharedTypes/navigation';
+import {MapPreview} from './MapPreview.tsx';
+import {ObservationList} from './ObservationList.tsx';
+import {ErrorBottomSheet} from '../../sharedComponents/ErrorBottomSheet.tsx';
+import {ActionButtons} from '../../sharedComponents/ActionButtons.tsx';
+import {ScreenContentWithDock} from '../../sharedComponents/ScreenContentWithDock.tsx';
+import {NativeStackNavigationOptions} from '@react-navigation/native-stack';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.Track.title',
+    defaultMessage: 'Track',
+    description:
+      'Title of track screen showing (non-editable) view of observation with map',
+  },
+  deleteTitle: {
+    id: 'screens.Track.deleteTitle',
+    defaultMessage: 'Delete track?',
+    description: 'Title of dialog asking confirmation to delete a track',
+  },
+  tracks: {
+    id: 'screens.Track.tracks',
+    defaultMessage: 'Tracks',
+  },
+});
+
+export const TrackScreen: NativeNavigationComponent<'Track'> = ({
+  route,
+  navigation,
+}) => {
+  const isMine = false;
+  React.useLayoutEffect(() => {
+    navigation.setOptions({
+      headerRight: isMine
+        ? () => (
+            <TouchableOpacity
+              hitSlop={{top: 16, right: 16, bottom: 16, left: 16}}
```

Comment:
nit: extract to constant

Reply:
https://github.com/digidem/comapeo-mobile/pull/412/commits/ec523495b19e5212d2c9ad78e55cf87ef7de6450